docpages/example_code/editing_messages.cpp

```diff
@@ -46,9 +44,38 @@ int main() {
 				/* change the channel name and edit the channel itself */
 				channel.set_name(name);
 				bot.channel_edit(channel);
-
 				event.reply("Channel name is now `" + name + "`.");
 			});
+		} else if (event.command.get_command_name() == "embed-send") {
+			dpp::embed embed = dpp::embed()
+				.set_color(dpp::colors::sti_blue)
+				.set_title("like and subscribe")
+				.set_url("https://dpp.dev/")
+				.set_author("Some author", "https://dpp.dev/", "https://dpp.dev/DPP-Logo.png")
+				.set_description("Creator is <creator name>");
+			event.reply(embed);
+		} else if (event.command.get_command_name() == "embed-edit") {
+			const auto description = std::get<std::string>(event.get_parameter("desc"));
+
+			/* get message to edit its embed after */
+			const dpp::snowflake msg_id = std::get<std::string>(event.get_parameter("msg-id"));
+			/* here string will automatically be converted to snowflake */
```

Comment:
If you're going to switch between putting the comments after the line (which btw nobody does, please stop doing this) and putting it before the line, stick to one.

You put it after here, then on line 71 you put the comment before.

Reply:
In that case I'll also have to edit the line 18 which, by the way, was approved in the last PR

Reply:
> In that case I'll also have to edit the line 18 which, by the way, was approved in the last PR

shouldn't have been, it makes no sense.